LPS-115035 Allow OIDC providers to be configured at system level only

stian-sigvartsen opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

stian-sigvartsen commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/5749/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5749/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 7 out of 20 jobs passed</h3><h3>:x: ci:test:relevant - 15 out of 43 jobs passed in 1 hour 32 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/98207db01962e9f9a9bfa4ffd11c98b61d2d7412">98207db01962e9f9a9bfa4ffd11c98b61d2d7412</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/6781f47bc448a9efe3c4df1eac5c1b8c4fc86c4e">6781f47bc448a9efe3c4df1eac5c1b8c4fc86c4e</a></p><details><summary><strong>ci:test:stable - 7 out of 20 jobs PASSED</strong></summary><div><h4>13 Failed Jobs:</h4><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400592/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323872/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400603/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400594/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323865/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323871/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323870/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400602/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323874/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400606/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289356/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392675/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400605/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</strong></strike></a></li></ul></div><details><summary><strong>7 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400604/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289358/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289353/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289357/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289351/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289354/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323873/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 15 out of 43 jobs PASSED</strong></summary><div><h4>28 Failed Jobs:</h4><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/5749/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289355/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400592/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400598/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289350/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420776/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/10</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400590/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/11</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400597/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400600/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289352/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400599/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323868/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400596/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400595/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/8</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289348/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/9</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323872/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400603/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400594/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323865/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323871/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323870/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400602/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323874/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400606/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289356/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392675/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400605/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323866/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</strong></strike></a></li></ul></div><details><summary><strong>15 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289349/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400601/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400591/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392676/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/364831/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400604/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400593/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323869/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289358/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289353/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289357/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289351/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323867/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289354/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323873/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><h5>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5749/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289355/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></h5><div><h6>Job Results:</h6><p>2 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/289355//consoleText">AXIS_VARIABLE=0,label_exp=!master #289355</a><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/289355//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
java.lang.NullPointerException
java.lang.NullPointerException
	at java.base/java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
	at java.base/java.util.concurrent.ConcurrentHashMap.putIfAbsent(ConcurrentHashMap.java:1541)
	at com.liferay.portal.security.sso.openid.connect.internal.OpenIdConnectProviderRegistryImpl.getOpenIdConnectProviderNames(OpenIdConnectProviderRegistryImpl.java:124)
	at com.liferay.login.authentication.openid.connect.web.internal.servlet.taglib.OpenIdConnectNavigationPreJSPDynamicInclude.include(OpenIdConnectNavigationPreJSPDynamicInclude.java:58)
	at com.liferay.portal.kernel.servlet.taglib.DynamicIncludeUtil.include(DynamicIncludeUtil.java:77)
	at com.liferay.taglib.util.DynamicIncludeTag.doEndTag(DynamicIncludeTag.java:32)
	at org.apache.jsp.login_jsp._jspx_meth_liferay$1util_dynamic$1include_2(login_jsp.java:1626)
	at org.apache.jsp.login_jsp._jspService(login_jsp.java:1154)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:411)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:473)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:377)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at com.liferay.portal.osgi.web.servlet.jsp.compiler.internal.JspServlet.service(JspServl...</code></pre></div></li><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/289355//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmoke_Smoke_">LocalFile.PortalSmoke#Smoke</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-smoke-tomcat90-mysql57-jdk11_open/0/0/LocalFile.PortalSmoke_Smoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-smoke-tomcat90-mysql57-jdk11_open/0/0/LocalFile.PortalSmoke_Smoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-smoke-tomcat90-mysql57-jdk11_open/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400592/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></h5><div><h6>Job Results:</h6><p>2 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400592//consoleText">AXIS_VARIABLE=0,label_exp=!master #400592</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400592//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a></div></li><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400592//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmoke_Smoke_">LocalFile.PortalSmoke#Smoke</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-smoke-tomcat90-mysql57-jdk8_stable/0/0/LocalFile.PortalSmoke_Smoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-smoke-tomcat90-mysql57-jdk8_stable/0/0/LocalFile.PortalSmoke_Smoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-smoke-tomcat90-mysql57-jdk8_stable/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400598/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400598//consoleText">AXIS_VARIABLE=0,label_exp=!master #400598</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400598//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive730_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive730</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive730/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/0/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive730/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289350/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/289350//consoleText">AXIS_VARIABLE=0,label_exp=!master #289350</a><ol><li><div><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/289350//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive621021_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive621021</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/1/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive621021/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/1/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive621021/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/1/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/420776/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/10</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/420776//consoleText">AXIS_VARIABLE=0,label_exp=!master #420776</a><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/420776//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchiveAutoUpgrade71103_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchiveAutoUpgrade71103</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/10/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchiveAutoUpgrade71103/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/10/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchiveAutoUpgrade71103/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1591720897004/test-portal-acceptance-pullrequest(master)/5749/functional-upgrade-tomcat90-mysql57-jdk8/10/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li>...</li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/5749/">test-portal-acceptance-pullrequest(master)#5749</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5749/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/csierra/liferay-portal/pull/735">csierra#735</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/12/Release/896.aspx">/Liferay DXP 7.3/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/12/Build/7831.aspx">test-1-8/test-portal-acceptance-pullrequest(master)/5749 - 2020-06-09[09:41:37]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-6.liferay.com/job/publish-spira-report/682/">publish-spira-report#682</a><br />

stian-sigvartsen commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/7475/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7475/jenkins-report.html">here</a>.</b></p>

stian-sigvartsen commented:
@csierra . I've removed that unlikely race condition for completeness and fixed a NPE. If you like the changes maybe we should squash these two new commits into the two last commits before sending as the final delta is small, but the commits are not as small.

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 20 out of 20 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 43 out of 43 jobs passed in 1 hour 22 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f8187ec441d1c41ad74dfb53a4713449e5a586e1">f8187ec441d1c41ad74dfb53a4713449e5a586e1</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/6781f47bc448a9efe3c4df1eac5c1b8c4fc86c4e">6781f47bc448a9efe3c4df1eac5c1b8c4fc86c4e</a></p><details><summary><strong>ci:test:stable - 20 out of 20 jobs PASSED</strong></summary><details><summary><strong>20 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398739/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/365309/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402539/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395192/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/374089/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395196/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/365308/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342559/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395197/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276286/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/363855/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393086/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342560/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276285/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395198/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/374092/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398747/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276283/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/363854/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393087/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 43 out of 43 jobs PASSED</strong></summary><details><summary><strong>43 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/7475/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267272/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398746/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398739/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/412281/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392546/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342561/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402541/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402537/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393084/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402538/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/397936/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398744/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276282/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/397934/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/363851/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/365309/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402539/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395192/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/374089/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395196/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/365308/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342559/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395197/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276286/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/363855/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393086/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342560/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401724/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398743/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398738/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/365310/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276287/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276285/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398740/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395195/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/395198/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/374092/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/398747/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/276283/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267273/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/363854/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/393087/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7475/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/7475/">test-portal-acceptance-pullrequest(master)#7475</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7475/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/csierra/liferay-portal/pull/735">csierra#735</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/12/Release/896.aspx">/Liferay DXP 7.3/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/12/Build/8041.aspx">test-1-3/test-portal-acceptance-pullrequest(master)/7475 - 2020-06-10[02:55:48]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-1.liferay.com/job/publish-spira-report/1390/">publish-spira-report#1390</a><br />
